test/Microsoft.TestPlatform.AcceptanceTests/ProcessTests.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+using Microsoft.TestPlatform.TestUtilities;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.TestPlatform.AcceptanceTests;
+
+[TestClass]
+public class ProcessTests : AcceptanceTestBase
+{
+    [TestMethod]
+    [NetCoreTargetFrameworkDataSource]
+    public void MissingFrameworkTextIsCorrectlyDisplayed(RunnerInfo runnerInfo)
```

Comment:
```suggestion
    public void WhenTestHostProcessExitsBecauseTheTargetedRuntimeIsNoFoundThenTheMessageIsCapturedFromTheErrorOutput(RunnerInfo runnerInfo)
```

Might be also worth adding some more info about the problem we were solving.